Makefile

```diff
@@ -0,0 +1,60 @@
+.PHONY: dummy_translations extract_translations fake_translations help pull_translations push_translations \
```

Comment:
Try to avoid backslashes at ends of lines.

Reply:
It was a typo. Resolved.